### ISSUE TITLE
Fork this site and make one for "Good first issues" in the Julia ecosystem

### ISSUE THREAD

**Issue by logankilpatrick** [JuliaCommunity/your-first-julia-pr#8]
@quinnj  @bkamins we could use this repo. CC @SaschaMann do you think we can just fork this and switch the labels we are querying for?

**Comment by logankilpatrick** [comment]
I also invited you both to this org ^

**Comment by bkamins** [comment]
What would be the preferred workflow? I have never seen this repo before.

**Comment by SaschaMann** [comment]
> CC @SaschaMann do you think we can just fork this and switch the labels we are querying for? 

Yes, just change the search query in https://github.com/JuliaCommunity/your-first-julia-pr/blob/main/main.go#L64

You can use this repo too, there's no need for anything hacktoberfest specific if there's a general site thst let's you find issues to work on.

**Comment by logankilpatrick** [comment]
I like having different sites specifically for the unique URL's (though this one should likely change to hacktoberfest)
